### PR TITLE
[fix bug 1319244] Remove search test code

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Removes all the bits and pieces relating to the cross-site search testing. 

## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1319244

## Checklist
- [ ] Requires l10n changes.
- [ ] Related functional & integration tests passing.

### CODE REVIEW

File: bedrock/base/templates/macros.html [hunk 1]
```diff
@@ -474,27 +474,3 @@ <h2 class="thank-you hidden">{{ _('Your download link was sent.') }}</h2>
     </footer>
   </section>
 {%- endmacro %}
-
-
-{% macro google_custom_search(search_type, type_class, default_term, term_list) %}
```
Comment (alexgibson) [inline bedrock/base/templates/macros.html]: I seem to remember some warning from either @pmac or @jgmize about removing a shared macro sometimes causing issues with pages that are still cached. Not sure if this is still the case, but flagging here?

**Comment by alexgibson** [comment]
There still seem to be some google related things left in the CSP white list, I'm guessing we should probably remove those too?

**Comment by craigcook** [comment]
D'oh, good catch on the CSP bits. Updated.

Not sure what to do about removing the macro... if removing it breaks things cached pages, does leaving it break things on uncached pages?

File: bedrock/base/templates/macros.html (hunk 1 above)
Comment (pmac) [inline bedrock/base/templates/macros.html]: Hmm... I don't remember that. Cached in what way? We should be able to remove any macro as long as it's no longer used by any templates.

**Comment by pmac** [comment]
This looks good to me. If there are issues I believe our selenium tests should catch them.

File: bedrock/base/templates/macros.html (hunk 1 above)
Comment (alexgibson) [inline bedrock/base/templates/macros.html]: All I remember was being asked to remove the macro in a second commit later on for fear of some kind of temporary breakage on deployment. Perhaps this is not the same thing.

**Comment by alexgibson** [comment]
Looks like this needs a rebase, bot otherwise r+ from me

**Pull request merged**